### DISCUSSION TITLE
v1.5.2-beta

### DISCUSSION THREAD

**Discussion by albertpb** [albertpb/sd-manager#16] [General]
- Fix workers import path

<hr /><em>This discussion was created from the release <a href='https://github.com/albertpb/sd-manager/releases/tag/v1.5.2'>v1.5.2-beta</a>.</em>

**Comment by AsuraAce** [comment]
seems to be a nice update:

- had to delete appdata to get 1.5.2 to start
- SD Manager seems to be almost instantly responsive on startup
- image import is fast
- image import could use an overall progress bar. or additional information to let the user know how long it takes
- image import: thumbnail creation progress bar doesn´t seem to update properly (stuck 0.01%). may have crashed, will let it run a little bit longer and let you know later 
- update to row / zoom level is very nice, later down the line it would probably be nice to let the user change the size (slider)
- tagging lora and checkpoints in their respective tabs doesn't seem to be necessary,

will check the rest out after the import is finished.

**Comment by AsuraAce** [comment]
yep, still stuck trying to generate thumbnails. not sure maybe this problem is caused because there are already some generated by SD Manager? 

also, clicking on a checkpoint that is unknown, leads to a black screen i can't recover from.

![image](https://github.com/albertpb/sd-manager/assets/13394741/183d2c81-9aca-4724-895a-dfde3811bdc9)

![image](https://github.com/albertpb/sd-manager/assets/13394741/30aed6fc-9129-4fe0-83af-b32ca3ada0d3)

**Reply by albertpb** [reply]
I forgot to remove a close call in the thumbnails worker thread. Try with v1.5.3

**Reply by albertpb** [reply]
For now I'm preventing to click on models with no data.
